ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -277,7 +279,12 @@ public boolean hasNext() {
         String responseLine = responseScanner.nextLine().trim();
         if (!responseLine.isEmpty()) {
           try {
-            bufferedRow = objectMapper.readValue(responseLine, StreamedRow.class);
+            GenericRow genericRow = new JsonUtil().buildGenericRowFromJson(responseLine);
```

Comment:
why are we no deserializing as a StreamedRow here anymore?
why are we instantiating a util class?
what is the util class creating an `ObjectMapper`, (which are expensive to create), for each row, rather than using the `objectMapper` field?

Reply:
We are still generating StreamRow from the GenericRow.
I updated the util class and made the method static along with a single static ObjectMapper that will be used y all method calls.